xml/System.Text/Encoding.xml

```diff
@@ -3902,7 +3902,7 @@ In .NET 5.0 and later versions, the code page name `utf-7` is not supported.
  This method returns a list of supported encodings, uniquely distinguished by code page. For a table that lists the supported encodings, see <xref:System.Text.Encoding>.
 
 > [!NOTE]
-> The list of supported encodings returned by the <xref:System.Text.Encoding.GetEncodings%2A> method does not include any additional encodings made available by any <xref:System.Text.EncodingProvider> implementations that were registered by calls to the <xref:System.Text.Encoding.RegisterProvider%2A> method.
+> The list of supported encodings returned by the <xref:System.Text.Encoding.GetEncodings%2A> will include any additional encodings made available by any <xref:System.Text.EncodingProvider> implementations that were registered by calls to the <xref:System.Text.Encoding.RegisterProvider%2A> method.
```

Comment:
Is this a new type? If it is not, and it existed in older .NET versions, was this the behavior (not including additional encodings)?

I ask in case the note needs to be expanded so that the description gets split between Framework and newer versions where the behavior is the opposite.

Reply:
The type was there but we have introduced EncodingProvider.GetEncodings which changed the behavior of Encoding.GetEncodings when there is a registered providers. yes the new behavior is new for .NET 5.0 and not for .NET Framework nor net core previous versions. We may mention that in this comment

Reply:
please suggest what you want to add here.